### PR TITLE
Fix test name typo

### PR DESCRIPTION
**Pull request by heliuchuan**

### CODE REVIEW

File: tests/e2e/api/transaction_submission.test.ts [hunk 1]
```diff
@@ -93,7 +93,7 @@ describe("transaction submission", () => {
       expect(desirializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
```
Comment (xbtmatt) [inline tests/e2e/api/transaction_submission.test.ts:93]:
```suggestion
      expect(deserializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
```

File: tests/e2e/api/transaction_submission.test.ts [hunk 2]
```diff
@@ -90,10 +90,10 @@ describe("transaction submission", () => {
       const deserializer = new Deserializer(rawTxn.rawTransaction);
       const desirializedTransaction = RawTransaction.deserialize(deserializer);
       expect(desirializedTransaction instanceof RawTransaction).toBeTruthy();
```
Comment (xbtmatt) [inline tests/e2e/api/transaction_submission.test.ts:92]:
```suggestion
      expect(deserializedTransaction instanceof RawTransaction).toBeTruthy();
```

File: tests/e2e/api/transaction_submission.test.ts [hunk 3]
```diff
@@ -90,10 +90,10 @@ describe("transaction submission", () => {
       const deserializer = new Deserializer(rawTxn.rawTransaction);
       const desirializedTransaction = RawTransaction.deserialize(deserializer);
```
Comment (xbtmatt) [inline tests/e2e/api/transaction_submission.test.ts:91]:
```suggestion
      const deserializedTransaction = RawTransaction.deserialize(deserializer);
```

File: tests/e2e/api/transaction_submission.test.ts [hunk 4]
```diff
@@ -88,12 +88,12 @@ describe("transaction submission", () => {
       });
       expect(rawTxn.rawTransaction instanceof Uint8Array).toBeTruthy();
       const deserializer = new Deserializer(rawTxn.rawTransaction);
-      const desirializedTransaction = RawTransaction.deserialize(deserializer);
-      expect(desirializedTransaction instanceof RawTransaction).toBeTruthy();
-      expect(desirializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
+      const deserializedTransaction = RawTransaction.deserialize(deserializer);
+      expect(deserializedTransaction instanceof RawTransaction).toBeTruthy();
+      expect(deserializedTransaction.payload instanceof TransactionPayloadEntryFunction).toBeTruthy();
```
Comment (gregnazario) [inline tests/e2e/api/transaction_submission.test.ts:93]: We can either fix all these misspellings now (this is copied a lot of places), or come back on a pass Friday, where we cleanup all mispelling.

**Review by gregnazario — APPROVED** [review]

**Pull request merged**